src/Robo/Commands/Setup/ToggleModulesCommand.php

```diff
@@ -69,10 +75,15 @@ protected function doToggleModules($command, $config_key) {
         ->drush("$command $modules_list")
         ->run();
       $exit_code = $result->getExitCode();
+      $eventInfo['active'] = TRUE;
+      $eventInfo['modules'] = md5($modules_list);
```

Comment:
Why would we send an md5 hash to amplitude?

Reply:
I want to know if the modules list has been changed from the BLT-provided defaults, or if it matches a few other common scenarios (i.e. if it's empty). But I don't want to send module names that could be potentially sensitive / identifiable.